package.json

```diff
@@ -26,7 +26,8 @@
     "docker-stop:dev": "docker-compose -f docker-compose.dev.yaml stop",
     "chain-setup": "tsx ./test-setup/main.ts",
     "clean": "rm -Rf dist",
-    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
```

Comment:
Do we not have prettier in here?

Reply:
Apparently not. We should address at some point & standardize on eslint only for coding issues, and prettier for format. But I wanted to keep that out of the scope of this PR.